src/Promitor.Core.Scraping/Configuration/Serialization/v1/Providers/MySqlDeserializer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Extensions.Logging;
+using Promitor.Core.Contracts.ResourceTypes.Enums;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model.ResourceTypes;
+
+namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Providers
+{
+    public class MySqlDeserializer : ResourceDeserializer<MySqlResourceV1>
+    {
+        public MySqlDeserializer(ILogger<MySqlDeserializer> logger) : base(logger)
+        {
+            Map(resource => resource.ServerName)
+                .IsRequired();
+            
+            // Not marking as optional for backwards compatibility
+            // 'Single' should be the default
```

Comment:
Don't think we have backwards compatability issues here, no? So let's make it required.

Reply:
Just double checking, do you mean like this?

````c#
Map(resource => resource.Type)
    .WithDefault(MySqlServerType.Single)
    .IsRequired();
````

Reply:
Since it's required, you can leave out the default value